inference-engine/cmake/share/InferenceEngineConfig.cmake.in

```diff
@@ -160,7 +160,7 @@ else()
 
         set(InferenceEngine_INCLUDE_DIRS ${IE_INCLUDE_DIR})
         set(InferenceEngine_LIBRARIES IE::inference_engine_legacy IE::inference_engine
-                                      IE::inference_engine_c_api)
+                                      IE::inference_engine_c_api IE::ngraph)
```

Comment:
as suggested in IM:
```
find_package(ngraph REQUIRED)
set_target_properties(IE::inference_engine PROPERTIES INTERFACE_LINK_LIBRARIES ngraph::ngraph)
```
This is needed explicitly, because currently this file does not use approach when targets are exported via cmake and dependencies between targets are kept.